pyccel/ast/headers.py

```diff
@@ -163,7 +182,7 @@ def kind(self):
     def is_static(self):
         return self._args[4]
 
-    def create_definition(self):
+    def create_definition(self, templates = []):
```

Comment:
You should be able to use a tuple to get rid of the codacy warning. Default arguments are static objects so they raise errors if they are modifiable. This is because if I do:
```
def f(a = []):
   print(a)
   a.append(1)

f()
f()
f()
```
The output will be:
```
[]
[1]
[1, 1]
```